syntax = "proto3";

import "trade/vo/HoldsVo.proto";
option java_package = "com.huasheng.quant.open.sdk.protobuf.trade.response";
option java_outer_classname = "StockQueryHoldsListResponseProto";
message StockQueryHoldsListResponse {
	repeated HoldsVo holdsList = 1;
}
